tests/acceptance/test_full_long_covid_study.py

```diff
@@ -34,7 +35,7 @@ class Cohort:
     # Patients registered on 2020-11-01
     _registrations = table("practice_registrations").date_in_range(index_date)
```

Comment:
Does this need something to make it a single row? i.e. some form of `first_by` or `last_by` operation. I guess it works as is at the moment because the data happens to have only one matching row, but at some point we'll want to start enforcing this in the cohort definition.

Reply:
Yes - but I think I've already fixed it in one of the other PRs

Reply:
No, actually I'd fixed it on this branch but forgot to push.  I need to fix some conflicts anyway.